apps/wallet/src/ui/app/pages/home/tokens/CoinActivityCard.tsx

```diff
@@ -37,7 +37,7 @@ export function CoinActivitiesCard({ coinType }: { coinType: string }) {
             <div className="p-2">
                 <Alert mode="warning">
                     <div className="font-semibold">
-                        {error?.message || 'Something went wrong'}
+                        {(error as Error).message}
```

Comment:
no need to fix this in this PR, but I imagine it would be better to type the Error field in our `useQuery` call inside `useGetTransactionsByAddress` so we don't have to manually cast `error`